nixos/modules/services/audio/goxlr-utility.nix

```diff
@@ -28,19 +28,29 @@ with lib;
     };
   };
 
-  config = mkIf config.services.goxlr-utility.enable
-    {
+  config =
+    let
+      goxlr-autostart = pkgs.stdenv.mkDerivation {
+        name = "autostart-goxlr-daemon";
+        priority = 5;
+
+        buildCommand = ''
+          mkdir -p $out/etc/xdg/autostart
+          cp ${cfg.package}/share/applications/goxlr-utility.desktop $out/etc/xdg/autostart/goxlr-daemon.desktop
+          chmod +rw $out/etc/xdg/autostart/goxlr-daemon.desktop
```

Comment:
```suggestion
          chmod +w $out/etc/xdg/autostart/goxlr-daemon.desktop
```

Reply:
This was straight from https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/make-startupitem/default.nix#L25 but seems unlikely that the read bit wouldn't already be set so I don't mind the change.